solidity/contracts/ECDSARewards.sol

```diff
@@ -88,8 +88,24 @@ contract ECDSARewards is Rewards {
     uint256 internal constant minimumECDSAKeepsPerInterval = 1000;
 
     BondedECDSAKeepFactory factory;
+    TokenStaking tokenStaking;
 
-    constructor(address _token, address payable _factoryAddress)
+    // The amount of tokens each individual beneficiary address
+    // can receive in a single interval is capped.
+    // TODO: set actual value
+    uint256 internal constant beneficiaryRewardCap = 400000 * 10**18;
+    // The total amount of rewards allocated to the given beneficiary address,
+    // in the given interval.
+    // `allocatedRewards[beneficiary][interval] -> amount`
+    mapping(address => mapping(uint256 => uint256)) allocatedRewards;
+    // The amount of interval rewards withdrawn to the given beneficiary.
+    mapping(address => mapping(uint256 => uint256)) withdrawnRewards;
```

Comment:
Should we add two public functions allowing to fetch this information?

Reply:
Added, plus another one for fetching the withdrawable amount